server/core/mail.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
       subject: `${opts.subject} - ${WIKI.config.title}`,
       text: opts.text,
       html: _.get(this.templates, opts.template)({
-        logo: WIKI.config.logoUrl,
+        logo: WIKI.config.host + WIKI.config.logoUrl,
```

Comment:
logoUrl can also be an absolute URL. There should be a check to see whether logoUrl starts with http and only prefix the host if not.